src/mozanalysis/config.py

```diff
@@ -2,7 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from typing import Optional
+from typing import List, Optional
```

Comment:
I was going to suggest we use generics for type hints instead of importing `List`, but apparently `mozanalysis` is on python 3.8 so maybe that's the more pressing issue...